configs/tool_resources.config

```
@@ -5,58 +5,151 @@ process {
         cpus       = { Math.min(6, consensus instanceof List ? consensus.size() : 1 ) }
     }
 
-    // EVALUATE_RAW_ASSEMBLY
-    withName: 'EVALUATE_RAW_ASSEMBLY:BUSCO' {
-        time       = 4.d
+    // EVALUATE ASSEMBLIES 
+    withName: 'BUSCO' {
+        time   = { input.meta.genome_size > 3000000000 ? 4.d : 2.d }
```

```suggestion
        time   = { input.meta.genome_size > 3000000000 ? 4.d : 2.d }
```
Can you check you're accessing the correct value here. I don't think there's one called `input.meta.genome_size`. 
It should be `meta.sample.genome_size` I think.